crates/cli/README.md

```
@@ -0,0 +1,222 @@
+# Set max threads
+cargo run -p khronos_cli -- --max-threads 4
+
+# Disable task library
+cargo run -p khronos_cli -- --disable-task-lib
+```
+
+## 🏗️ Architecture
+
+The CLI is built with a clean, modular architecture:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)